### PR TITLE
First pass at a dockerfile for alma_8 development image for icetray/main

### PR DESCRIPTION
**Pull request by blaufuss**
Add support for almalinux 8.4 based dev platform

### CODE REVIEW

**Comment by blaufuss** [comment]
@dsschult any concerns with this?

File: almalinux8/icetray/main/Dockerfile_base-devel [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+RUN ln -s /usr/local/etc/OpenCL/vendors/pocl.icd /etc/OpenCL/vendors 
+
+RUN wget https://github.com/Starlink/pal/releases/download/v0.9.8/pal-0.9.8.tar.gz && \
+	tar zxf pal-0.9.8.tar.gz && cd pal-0.9.8 && \
+	./configure --prefix=/usr/local --without-starlink && make install && cd
+
+#RUN dnf config-manager --add-repo \
+#	https://repositories.intel.com/graphics/rhel/8.3/intel-graphics.repo
+#UN dnf install -y intel-opencl intel-media intel-mediasdk level-zero intel-level-zero-gpu
+#RUN dnf install -y intel-igc-opencl-devel intel-igc-cm level-zero-devel
```
Comment (dsschult) [inline almalinux8/icetray/main/Dockerfile_base-devel:37]: Do we need this commented out code?

**Comment by dsschult** [comment]
No real concerns, other than we should figure out how to put these docker file builds into the main icetray repo so they are triggered automatically (or some other method of triggering from the main repo).  But that's probably broader than this PR.

File: almalinux8/icetray/main/Dockerfile_base-devel (hunk 1 above)
Comment (blaufuss) [reply almalinux8/icetray/main/Dockerfile_base-devel:37]: I can clean it up

Comment (blaufuss) [reply almalinux8/icetray/main/Dockerfile_base-devel]: done, closing the PR

**Pull request merged**